src/components/Shared/MultiPageLayout/MultiPageMenu/MultiPageMenuItems.ts

```diff
@@ -44,6 +44,10 @@ export const reportNavItems: NavItems[] = [
     id: 'coaching',
     title: i18n.t('Coaching'),
   },
+  {
```

Comment:
Please comment this out or revert it. We'll wait to add the links until the expense report is fully done and has been QAed.